tracer/samples/AutomaticTraceIdInjection/RunSampleWithAutomaticInstrumentation.sh

```diff
@@ -0,0 +1,40 @@
+## Set the project you want to test
+PROJECT=Log4NetExample
+APP_PATH=./bin/Debug/net7.0
+
+## Setup automatic instrumentation
+## cf https://www.nuget.org/packages/Datadog.Trace.Bundle#readme-body-tab
+## Note that the agent must be configured if you want to receive the traces
+
+## .NET Core
+export CORECLR_ENABLE_PROFILING=1
+export CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+## Replace this path by the appropriate one depending on your system (cf https://www.nuget.org/packages/Datadog.Trace.Bundle#readme-body-tab)
+export CORECLR_PROFILER_PATH=$APP_PATH/datadog/osx/Datadog.Trace.ClrProfiler.Native.dylib
```

Comment:
given this is a bash-script we could (should?) probably auto-detect the platform and set this as appropriate?

Reply:
I've removed this in favor of `BuildAndRunSample.ps1`, but only have Windows/Linux as supported platforms (others could be added easily though to it if that makes sense)